app/api/api_v1/endpoints/document.py

```diff
@@ -66,27 +62,24 @@ def update_document( collection_name: str, path: str, document: schemas.Document
     """
     collection = get_collection_by_name(collection_name)
 
-    for current_document in collection.documents:
-        if current_document.path == path:
-            current_document.localeCode = document.localeCode
-            current_document.title = document.title
-            current_document.description = document.description
-            current_document.render = document.render
-            return current_document
+    if path in collection.documents:
```

Comment:
A bit stange, why do you copy it field by filed ?

Reply:
To ensure user doesn't accidentally reassign the path (path is excluded from other variables). Since path serves also as an index, it's better not to meddle  with. The implementation is not perfect though.